caffe2/core/export_caffe2_op_to_c10.h

```diff
@@ -74,13 +75,21 @@ inline void _call_caffe2_op_from_c10(
   } else {
     AT_ASSERT(preallocated_outputs.isTensorList());
     outputs = std::move(preallocated_outputs).toTensorList();
+    TORCH_INTERNAL_ASSERT(num_outputs == outputs.size());
```

Comment:
this assert is triggering with internal tests. Looking for a repro unless you know why.

Reply:
I've changed it to assign `num_outputs = outputs.size()` to avoid any problems. I think it must be happening when the operator returns a `TensorList` so `num_outputs` isn't encoded in the schema.